scripts/travis/configure_dev.sh

```diff
@@ -11,9 +11,7 @@ function installGo() {
     echo "Install Go for arch ${OS_ARCH} with GO VERSION ${GO_VERSION} to replace ${INSTALLED_GO_VERSION}"
 
     if [[ "$INSTALLED_GO_VERSION" != "$GO_VERSION" ]]; then
-        echo "Correct Go version cannot be found; downloading go $GO_VERSION for ${OS_ARCH}..."
-        # go is not installed ?
-        # e.g. https://dl.google.com/go/go1.13.5.linux-arm64.tar.gz
+        echo "Correct Go version not be found; downloading go $GO_VERSION for ${OS_ARCH}..."
```

Comment:
I'm alittle bit puzzled here.. usually we fix typos, not introduce these intentinally.

Reply:
Not intentional, fixed.